deploy/deploy.json

```diff
@@ -4,7 +4,7 @@
   "repositories": [{
       "name": "avalon-core",
       "url": "https://github.com/pypeclub/avalon-core.git",
-      "branch": "2.7.0-pype"
+      "tag": "2.8.0-pype"
```

Comment:
Shall we change this to develop? Because for deployment in clients we use their deployment json.

Reply:
no I would keep this locked to stable versions. Think of people who just wanna give it a shot.